eth2util/keystore/load.go

```diff
@@ -0,0 +1,134 @@
+package keystore
+
+import (
+	"context"
+	"encoding/json"
+	"github.com/obolnetwork/charon/app/errors"
+	"github.com/obolnetwork/charon/app/forkjoin"
+	"github.com/obolnetwork/charon/app/z"
+	"github.com/obolnetwork/charon/tbls"
+	"os"
+	"path"
+	"path/filepath"
+	"regexp"
+	"strconv"
+)
+
+type KeyFiles []KeyFile
+
+// Keys returns the private keys in random order.
+func (s KeyFiles) Keys() []tbls.PrivateKey {
+	resp := make([]tbls.PrivateKey, len(s))
+	for _, ks := range s {
+		resp = append(resp, ks.PrivateKey)
+	}
+
+	return resp
+}
+
+// SequencedKeys returns the private keys in strict sequential file index order from 0 to N.
+// If the indexes are unknown or not sequential or there are duplicates, an error is returned.
+func (s KeyFiles) SequencedKeys() ([]tbls.PrivateKey, error) {
```

Comment:
what if there is a case where we have KeyFiles as: `keystore-1.json`, `keystore-5.json `and `keystore-17.json`. Length of keyfiles is 3 but indices are 1, 5 and 17. I think this function will panic in this case with out of bounds message.

Reply:
I added a test, it does not